tests/sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/test_baseline_no_first_session/fenix.first_session.schema.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- client_info:
+    client_id: client-2
```

Comment:
If I'm reading this right, a client who sends just a `first_session` ping won't have an entry in this table? I'm not sure if that's the desired behavior